src/components/Microsoft.DotNet.UpgradeAssistant.MSBuild/UpgraderMsBuildExtensions.cs

```diff
@@ -46,6 +47,11 @@ public static void AddMsBuild(this IServiceCollection services, Action<Workspace
             services.AddTransient<Func<MSBuildWorkspaceUpgradeContext>>(sp => () => sp.GetRequiredService<MSBuildWorkspaceUpgradeContext>());
         }
 
+        private static string GetCanonicalIncludePath(string path)
+        {
```

Comment:
This showed up in another place. Let's just keep a single copy somewhere.

Reply:
Fixed.